support/cas-server-support-oauth/src/test/java/org/apereo/cas/ticket/refreshtoken/OAuthRefreshTokenExpirationPolicyTest.java

```diff
@@ -0,0 +1,52 @@
+package org.apereo.cas.ticket.refreshtoken;
+
+import org.apereo.cas.authentication.Authentication;
+import org.apereo.cas.authentication.principal.Service;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.Assert.*;
+
+/**
+ * Adding test to check expired refresh token.
+ *
+ * @author Phillip Rower
+ * @since 5.0.0
+ */
+public class OAuthRefreshTokenExpirationPolicyTest {
+    @Mock
+    private Service mockService;
+
+    @Mock
+    private Authentication mockAuthentication;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void isExpiredTrueTest() {
+        final OAuthRefreshTokenExpirationPolicy policy = new OAuthRefreshTokenExpirationPolicy(-1L);
+        DefaultRefreshTokenFactory defaultRefreshTokenFactory = new DefaultRefreshTokenFactory();
+        defaultRefreshTokenFactory.setExpirationPolicy(policy);
+        final RefreshTokenImpl refreshToken = (RefreshTokenImpl) defaultRefreshTokenFactory.create(mockService, mockAuthentication );
```

Comment:
Do you need to cast down to the Impl class or can we stick with the parent interface? 

(Also, extra space at the end should be removed)